src/App.js

```diff
@@ -1,16 +1,44 @@
 import React from 'react';
 import './App.css';
+import { useState } from 'react';
```

Comment:
Small nit, since L1 and L3 both import classes/functions from react, you can combine them in the same line:
```
import React, { useState } from 'react';
```